src/mame/jaleco/jaleco_vj_pc.cpp

```diff
@@ -148,6 +151,12 @@ void jaleco_vj_pc_device::device_add_mconfig(machine_config &config)
 	serport1.cts_handler().set("board4:fdc37c93x", FUNC(fdc37c93x_device::ncts2_w));
 }
 
+void jaleco_vj_pc_device::device_config_complete()
+{
+	auto king_qtaro = m_king_qtaro.finder_target();
+	king_qtaro.first.subdevice<jaleco_vj_king_qtaro_device>(king_qtaro.second)->set_bus_master_space(m_maincpu, AS_PROGRAM);
```

Comment:
This looks pretty suspect – I would have thought bus mastering should be supported by the PCI framework to avoid needing to do things like this.

@galibert how are bus mastering PCI devices supposed to work their magic?